### PR TITLE
Logger set screen refactor

### PR DESCRIPTION
**Pull request by BCrlrd**
This refactors LoggerSetScreen.

For some reason I can't reach 100% line coverage on LoggerSetScreen, even though all lines/branches have been triggered. If anyone has any idea how to fix it, feel free to implement it or let me know.

< testing, organisation

### CODE REVIEW

File: src/main/java/nl/tudelft/semgroup4/LoggerSetScreen.java [hunk 1]
```diff
@@ -102,56 +61,48 @@ public LoggerSetScreen(ResourcesWrapper res, GameContainer container) throws Sli
...
+            typeFont.drawString(renderCoordX,
+                    renderCoordY + i * 50,
+                    text[i],
+                    controller.decideColor(severities[i]));
         }
     }
 
+    /**
+     * Get all mouse over areas (only used for testing).
+     * @param overArea
```
Comment (cathmer4) [inline src/main/java/nl/tudelft/semgroup4/LoggerSetScreen.java]: This method sets a mouseoverarea, and only the first one too. You should update your javadoc (or method) to reflect this.

File: src/main/java/nl/tudelft/semgroup4/LoggerSetScreenController.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+     *                  the font with which the buttons will be created
+     * @param res
+     *                  a resource wrapper which will create the image
+     * @return areas
+     *                  an array containing all created buttons
+     * @throws SlickException
+     *                  image can't be created
+     */
+    public MouseOverArea[] createMouseOverAreas(float overCoordX, float overCoordY,
+            GameContainer container, String[] text,TrueTypeFont typeFont,
```
Comment (cathmer4) [inline src/main/java/nl/tudelft/semgroup4/LoggerSetScreenController.java]: You should make this method protected.

File: src/main/java/nl/tudelft/semgroup4/LoggerSetScreen.java [hunk 3]
```diff
@@ -102,56 +61,48 @@ public LoggerSetScreen(ResourcesWrapper res, GameContainer container) throws Sli
...
+    /**
+     * Renders all the buttons.
+     *
+     * @param renderCoordX
+     *                  the x coordinate to use
+     * @param renderCoordY
+     *                  the first y coordinate to use
+     */
+    public void renderText(float renderCoordX, float renderCoordY) {
+        for (int i = 0; i < text.length; i++) {
```
Comment (cathmer4) [inline src/main/java/nl/tudelft/semgroup4/LoggerSetScreen.java]: This method can be made private.

**Comment by cathmer4** [comment]
The loggerscreen looks nice, it is a lot cleaner now. I just had a couple of small things I think you should change. 
Also, it would be nice if you could pause the OptionsState that lies underneath it, so you can't click on anything in it when the LoggerScreen is open.

File: src/main/java/nl/tudelft/semgroup4/LoggerSetScreen.java (hunk 1 above)
Comment (BCrlrd) [inline src/main/java/nl/tudelft/semgroup4/LoggerSetScreen.java]: Oh whoops, I changed what I wanted that method to do but forgot to change the javadoc. Will change!

File: src/main/java/nl/tudelft/semgroup4/LoggerSetScreenController.java (hunk 2 above)
Comment (BCrlrd) [inline src/main/java/nl/tudelft/semgroup4/LoggerSetScreenController.java]: Changed in next commit.

File: src/main/java/nl/tudelft/semgroup4/LoggerSetScreen.java (hunk 3 above)
Comment (BCrlrd) [inline src/main/java/nl/tudelft/semgroup4/LoggerSetScreen.java]: Changed, just checking if the tests still work. It was only public for testing purposes.

**Comment by BCrlrd** [comment]
I tried fixing that, but for some reason slick no longer accepts inputs on the LoggerSetScreen if I pause the OptionsState. If I can find a work around I will implement it, but for now inputs will work in the background.

**Comment by cathmer4** [comment]
It doesn't have a high priority, so I will merge this now.

**Pull request merged**